lib/selbot2/cla.rb

```diff
@@ -0,0 +1,27 @@
+require 'uri'
+
+module Selbot2
+  class CLA
+    include Cinch::Plugin
+
+    prefix Selbot2::PREFIX
+    match /(?:cla) (.+)/
+
+
+    def execute(message, query)
+      commits_url = "https://github.com/SeleniumHQ/selenium/commits?author=#{URI.escape query}"
+      resp   = RestClient.get(commits_url)
+      result = Nokogiri.XML(resp)
+
+      if result.css('.commit').any?
+        message.reply "#{query} has signed the CLA and has #{result.css('.commit').size} commits."
```

Comment:
right.. the only way that we can know if they signed it, is via how many commits they had.  so would you then prefer a different command directive? (in place of :cla)